lib/pbench/test/unit/agent/task/test_results_push.py

```diff
@@ -129,7 +128,8 @@ def test_server_and_relay():
 
     @staticmethod
     @responses.activate
-    def test_relay(relay_mock):
+    def test_relay():
+        __class__.relay_mock()
```

Comment:
I don't object to using `__class__`, per se (it looks ugly but I appreciate the flexibility it provides), but at line 191 (etc.) we used `TestResultsPush`, instead...and, I feel like we should use the same treatment for each of `server_mock()` and `relay_mock()`.

Reply:
Yeah, I noticed that when I looked at the diffs after I pushed the commit; if I decide to make another change, I'll probably change this to match.